src/representations/calculator_spherical_invariants.hh

```diff
@@ -38,6 +38,8 @@
 #include "rascal_utility.hh"
 #include "math/math_utils.hh"
 
+#include "wigxjpf.h"
```

Comment:
Should this not also be something with an `external` folder?

Reply:
json.hh is part of the repository, hence the external folder. This one is downloaded and compiled by cmake, so we don't control its path.